accounts/models.py

```diff
@@ -116,6 +116,15 @@ def get_complete_address(self):
             return f'{self.street_address1}, {self.street_address2}, {self.city} - {self.pincode}, {self.state}'
         return f'{self.street_address1}, {self.city} - {self.pincode}, {self.state}'
 
+    @property
+    def get_profile_image_url(self):
+        if self.profile_image and hasattr(self.profile_image, 'url'):
+            return self.profile_image.url
+        elif self.gender == 'M':
```

Comment:
Replace the second statement to `F` and put `man.png` in else statement.

Reply:
ok Sir I will fix it